infra/examples-dev/aws-all/misc-data-source-variables.tf

```diff
@@ -26,10 +26,10 @@ variable "jira_example_issue_id" {
   description = "If using Jira Server/Cloud connector, provide id of an issue for only to be used as part of example calls for Jira (ex: ETV-12)"
 }
 
-variable "github_api_host" {
+variable "github_enterprise_server_host" {
```

Comment:
this is a breaking change; so need to leave the current one `github_api_host` as deprecated, and add the new one. then take new one unless the default value; and if new one is the default value, take the old one.

Reply:
Ok; more or less. Despite the prev version where there is a default value (if `github_api_host` is not provided it will use the cloud url for github) here is nothing for that, so it will try to take old one, then new one and if not a placeholder for the host that could make easier to identity what's pending on settings